packages/core/src/app.ts

```diff
@@ -944,7 +944,9 @@ let d = w.define;
 {{/if}}
 
 {{#if autoRun ~}}
-  i("{{js-string-escape mainModule}}").default.create({{{json-stringify appConfig}}});
+  if (!window.disableAutoBoot) {
```

Comment:
We should likely bike-shed the name of this environment variable.

My first thought would be: `EMBER_DISABLE_AUTO_BOOT`

Also, should we make this `window`? Or simply leave it unbound, allowing the wrapping environment to provide it without mutating a potential global.

Reply:
Makes sense, I will update the code to use `EMBER_DISABLE_AUTO_BOOT`. Thank you

Reply:
> Also, should we make this `window`? Or simply leave it unbound, allowing the wrapping environment to provide it without mutating a potential global.

In other parts of the template `window` is used. But I agree we should pull from global context without binding to the window object. I will update the code to below

`typeof EMBER_DISABLE_AUTO_BOOT === 'undefined' || ! EMBER_DISABLE_AUTO_BOOT`